src/event.rs

```diff
@@ -185,20 +185,27 @@ where
 	pub(crate) fn next_event(&self) -> Arc<LdkLiteEvent> {
 		let locked_queue = self
 			.notifier
-			.wait_while(self.queue.lock().unwrap(), |queue| queue.0.is_empty())
+			.wait_while(self.queue.lock().unwrap(), |queue| queue.is_empty())
 			.unwrap();
-		Arc::clone(&locked_queue.0.front().unwrap())
+		Arc::clone(&locked_queue.front().unwrap())
 	}
 
 	pub(crate) fn event_handled(&self) -> Result<(), Error> {
 		{
 			let mut locked_queue = self.queue.lock().unwrap();
-			locked_queue.0.pop_front();
-			self.persister.persist(EVENTS_PERSISTENCE_KEY, &*locked_queue)?;
+			locked_queue.pop_front();
+			self.persist_queue(&*locked_queue)?;
 		}
 		self.notifier.notify_one();
 		Ok(())
 	}
+
+	fn persist_queue(&self, locked_queue: &VecDeque<Arc<Event>>) -> Result<(), Error> {
+		self.persister
+			.persist(EVENTS_PERSISTENCE_KEY, &EventQueueSerWrapper(locked_queue))
+			.map_err(|_| Error::PersistenceFailed)?;
+		Ok(())
```

Comment:
nit: can drop the `?; Ok(())`, I think

Reply:
Dropped with https://github.com/lightningdevkit/ldk-lite/pull/10/commits/c2ae588e21e11355344b207f7e040acde361cec0.